test/gtest/ucp/test_ucp_mmap.cc

```diff
@@ -214,6 +234,70 @@ void test_ucp_mmap::test_rkey_management(ucp_mem_h memh, bool is_dummy,
     ucp_rkey_buffer_release(rkey_buffer);
 }
 
+void test_ucp_mmap::compare_distance(const ucs_sys_dev_distance_t &dist1,
+                                     const ucs_sys_dev_distance_t &dist2)
+{
+    EXPECT_NEAR(dist1.bandwidth, dist2.bandwidth, 600e6); /* 600 MBs accuracy */
+    EXPECT_NEAR(dist1.latency, dist2.latency, 20e-9); /* 20 nsec accuracy */
+}
+
+void test_ucp_mmap::test_rkey_proto(ucp_mem_h memh)
```

Comment:
It is always called right after `test_rkey_management` (a repetitive pattern).

Maybe put it in a separate function (e.g something like `test_rkey()` which would call them both)?